### PR TITLE
Update Shelly documentation

### PR DESCRIPTION
**Pull request by bieniu**
## Proposed change

This PR adds information about `appliance type` and Shelly events and extends the list of issues and limitations.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42508 https://github.com/home-assistant/core/pull/43479
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by thecode — APPROVED** [review]

**Review by thecode — DISMISSED** [review]

File: source/_integrations/shelly.markdown [hunk 1]
```diff
@@ -51,8 +51,59 @@ Names are set from the device web page:
...
+  trigger:
+    platform: event
+    event_type: shelly.click
+    event_data:
+      device: shellyswitch25-AABBCC
+      channel: 1
+      click_type: single
+  action:
+    service: light.toggle
+    entity: light.living_room
```
Comment (frenck) [inline source/_integrations/shelly.markdown]: This should have been `entity_id`, right?
```suggestion
    entity_id: light.living_room
```

File: source/_integrations/shelly.markdown [hunk 2]
```diff
@@ -51,8 +51,59 @@ Names are set from the device web page:
...
+  trigger:
+    platform: event
+    event_type: shelly.click
+    event_data:
+      device: shellyswitch25-AABBCC
+      channel: 2
+      click_type: long
+  action:
+    service: light.toggle
+    entity: light.lamp_living_room
```
Comment (frenck) [inline source/_integrations/shelly.markdown]:
```suggestion
    entity_id: light.lamp_living_room
```

File: source/_integrations/shelly.markdown [hunk 3]
```diff
@@ -51,8 +51,59 @@ Names are set from the device web page:
...
+| `S`                | `single`      |
+| `SS`	             | `double`      |
+| `SSS`              | `triple`      |
+| `L`	               | `long`        |
+| `SL`	             | `single_long` |
+| `LS`	             | `long_single` |
+
+<div class="note">
+Not all devices support all input events. You can check on [Shelly API Reference](https://shelly-api-docs.shelly.cloud/) website what types of Shelly input events your device supports.
+</div>
```
Comment (frenck) [inline source/_integrations/shelly.markdown]: Needs a new-line in order to be able to switch between Markdown & HTML context.

```suggestion
<div class="note">

Not all devices support all input events. You can check on [Shelly API Reference](https://shelly-api-docs.shelly.cloud/) website what types of Shelly input events your device supports.

</div>
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/shelly.markdown (hunk 1 above)
Comment (bieniu) [reply source/_integrations/shelly.markdown]: Of course 🤦‍♂️

**Review by frenck — APPROVED** [review]
Thanks! 👍

File: source/_integrations/shelly.markdown [hunk 4]
```diff
@@ -51,8 +51,61 @@ Names are set from the device web page:
...
+  trigger:
+    platform: event
+    event_type: shelly.click
+    event_data:
+      device: shellyswitch25-AABBCC
+      channel: 2
+      click_type: long
+  action:
+    service: light.toggle
+    <span class="x x-first x-last">entity_id</span>: light.lamp_living_room
```
Comment (bieniu) [inline source/_integrations/shelly.markdown:87]: @frenck Something is wrong here after merging. I'll open a new PR to fix this.

Comment (frenck) [reply source/_integrations/shelly.markdown:87]: Wut? That looks like a GitHub Suggestion failure...

Comment (bieniu) [reply source/_integrations/shelly.markdown:87]: Yes, that's weird.

Comment (bieniu) [reply source/_integrations/shelly.markdown:87]: PR with the fix https://github.com/home-assistant/home-assistant.io/pull/15785

**Pull request merged**